setup.py

```diff
@@ -48,7 +48,7 @@ def get_version():
     packages=find_packages(exclude=['tests*']),
     include_package_data=True,
     zip_safe=False,
-    python_requires=">=3.6",
+    python_requires=">=3.9",
     install_requires=[
         'ansible>=2.5.0',
```

Comment:
While bumping the python version to minimum of 3.9, we probably should update the minimum version for ansible, as 2.5.0 is unmaintained [1]. When installing teflo, it's always going to be pulling in the latest ansible version which as of now is 7.0.0.

```suggestion
        'ansible>=7.0.0',
```

[1] https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#ansible-community-changelogs